new.go

```diff
@@ -25,8 +27,36 @@ const (
 	DefaultTransport = "docker://"
 )
 
+func reserveSELinuxLabels(store storage.Store, id string) error {
+
+	if selinux.GetEnabled() {
+		containers, err := store.Containers()
+		if err != nil {
+			return err
+		}
+
+		for _, c := range containers {
+			if id == c.ID {
+				continue
+			} else {
+				b, err := OpenBuilder(store, c.ID)
+				if err != nil {
+					if err == storage.ErrContainerUnknown {
+						continue
+					}
+					return err
+				}
+				// Prevent containers from using same MCS Label
+				if err := label.ReserveLabel(b.ProcessLabel); err != nil {
+					return err
+				}
+			}
+		}
+	}
+	return nil
+}
```

Comment:
Ok, maybe I've not had enough tea, but I'm struggling with this function.  Shouldn't we also be checking the ProcessLabel for 'id' rather than just continuing?  Is mcsList in selinux.go something that we can persist, or is it a map that we have to build every time we create a container?

Reply:
The idea is to go through all of the defined containers and reserve their MCS Labels.  Then when you create a new container we get a fresh unigue MCS Label.  I don't think persisting this is a great idea.  Eventually we need to coordinate with kpod/crio/buildah.

Reply:
If @nalind is hip with this, then I'm fine.  Maybe I'm just terribly misreading the code, but aren't we going through this code and touching every container every time we create a new container?  If so, it seems like a lot of overhead, but I'm not sure how to make it better.  That's the basis for my concern here fwiw.

Reply:
Yes you are right,  and we will have to do this for CRI-O also and kpod run (We do this on Docker now also.).  Unless we eventually move to some light weight database,  or start persisting a store of allocated MCS Labels, we will have to do this.

Reply:
Ok, thanks this might be the one place a BFD might be handy.  LW db sounds like an interesting side project, I'll try poking around at that.